packages/standalone-analyzer/src/assets.test.ts

```diff
@@ -0,0 +1,166 @@
+import {
```

Comment:
I added unit tests with mocks within this directory for the basics of the `standalone-analyzer` functionality. See `integration-tests` for actual calls to apis